crates/config/src/config.rs

```diff
@@ -187,49 +187,86 @@ impl Default for OffChainValidatorConfig {
 /// Config the base/minimal fee rules
 #[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
 pub struct FeeConfig {
-    supported_fee_sudt_ids: Vec<u32>,
-    /// known as gasPrice in Ethereum
+    /// HashMap<fee_sudt_id, fee_rate>
     ///
-    ///   denoted in shannons, which itself is a fractional denomination of CKBytes.
+    /// fee_rate is known as gasPrice in Ethereum.
+    ///
+    /// The defaut fee_sudt_id is 1, which is CKB, and the ckb_fee_rate is denoted in shannons,
+    /// which itself is a fractional denomination of CKBytes.
     ///   1 CKByte = 100,000,000 Shannons
-    fee_rate: u64,
+    fee_rates: HashMap<u32, u64>,
     meta_contract_fee_weight: u8,
     sudt_transfer_fee_weight: u8,
     withdraw_fee_weight: u8,
 }
+const DEFAULT_CKB_FEE_RATE: u64 = 500;
 impl FeeConfig {
     pub fn is_supported_sudt(&self, sudt_id: u32) -> bool {
-        self.supported_fee_sudt_ids.contains(&sudt_id)
+        self.fee_rates.contains_key(&sudt_id)
+    }
+    /// The defaut fee_sudt_id is 1, which is CKB
+    fn default_fee_rate(&self) -> u128 {
+        self.fee_rates
+            .get(&gw_common::builtins::CKB_SUDT_ACCOUNT_ID)
+            .unwrap_or(&DEFAULT_CKB_FEE_RATE)
+            .to_owned()
+            .into()
     }
     /// Get the minimal fee of meta contract
     pub fn meta_contract_base_fee(&self) -> u128 {
```

Comment:
Please pass sudt_id as argument to this method, we shouldn't using a `default` sudt to return fee